pkg/console/util.go

```diff
@@ -512,6 +515,25 @@ func doInstall(g *gocui.Gui, hvstConfig *config.HarvesterConfig, webhooks Render
 		env = append(env, fmt.Sprintf("HARVESTER_ADDITIONAL_KERNEL_ARGUMENTS=%s", hvstConfig.OS.AdditionalKernelArguments))
 	}
 
+	// when WipeAllDisks is enabled then find all non installation disks with COS_ prefixed labels
+	// and add them to a list for wiping
```

Comment:
From a UX perspective, we might need to document that this setting wipe all disks except for the installation and data disks.